package main

func main() {
	panic("Error!")
}
